### DISCUSSION TITLE
Knots in the mesh

### DISCUSSION THREAD

**Discussion by behroozkarami** [neperfepx/neper#431] [Q&A] [answered]
Dear All,

After meshing the tessellation I get a (2D) mesh, though correct, there exist some points look like knots in the mesh exhibiting a relatively large number of elements (with small sizes) created around those points which look not to be necessary.
Was wondering how can I resolve those knots?
For meshing I use the simple command: neper -M d.tess -elttype tri
Thank you very much

**Comment by rquey** [comment, marked as answer]
This can be avoided by switching regularization on in  -T (`-reg 1`) to remove the small edges, and if necessary, by increasing the value of the "progression factor" (`-pl 10` or more) in -M.
